### DISCUSSION TITLE
"SQL query is empty" despite not being empty

### DISCUSSION THREAD

**Discussion by fidel-perez** [supabase/supabase#2428] [Questions]
I am trying to run a SQL following [this ](https://supabase.io/docs/guides/with-vue-3)tutorial and I always get "SQL query is empty". 
I have tried with many other queries and all of them return the same result. 
How to make SQL queries work? Thank you!

![image](https://user-images.githubusercontent.com/4046326/126057253-17fe2238-b5e3-4499-98b8-aba467ce2e1b.png)

**Comment by burggraf** [comment]
This is an odd one!  A few things to try or check:

- Can you log out and log back into Supabase to make sure you are logged in and have a valid connection to the database?
- What happens when you try a simple query that doesn't need to touch the database, such as:  `select 1 as test_column;` Does that work and give a proper return value?
- Have you tried breaking the SQL statements up to run them individually?  Like running just one of the `create policy` SQL statements by itself?

**Reply by fidel-perez** [reply]
Thanks for your answer!
1: Yes, this happens accross several logins, in different days.
2: Same thing
3: Yes, any statement gives the same result.

After using more supabase, I am convinced this is not a particular problem with SQL, it is that the settings file associated to my account are corrupt and this is resulting in all sorts of unexpected behaviours. My account is barely functional.
I am thinking of making another one but I dont want to have to migrate my project, so I [reported this bug ](https://github.com/supabase/supabase/issues/2432) and hoping someone will have a look soon :)

Thanks again for your answer!

**Reply by burggraf** [reply]
Please let me know if creating a new PROJECT works for you, and if not try setting up a new ACCOUNT. Let me know what works and once we isolate the problem we can have someone here look at your project and/or account. Thanks!

**Reply by fidel-perez** [reply]
Thanks for your repply @burggraf .
I just tested with a just created account and It seems to be also bugged. In this case I cant even create a new project with it.
I updated the information on the bug ticket I created: https://github.com/supabase/supabase/issues/2432#issuecomment-883482108

Please check the comment, I rather not repeat the same information I posted there :)
